b3dm.py

```diff
@@ -73,7 +73,7 @@ def writeHeader(self, gltf_bin, num_feature_features, num_batch_features):
 		         len(gltf_bin)
 	
 		output = bytearray()
-		output.extend(B3DM_MAGIC)
+		output.extend(B3DM_MAGIC.encode('ascii'))
```

Comment:
It seems like you should probably just use UTF-8 since that is used elsewhere in the same file and doesn't change the result here.

Reply:
I considered `utf-8`, but my thought is that we know all the GLTF magic strings have to be ASCII characters, so we might as well enforce it here, and an exception thrown here means we're definitely not dealing with valid magic (same for decoding magic).

Reply:
Done in a2c2e00